python-sdk/tests/benchmark/Makefile

```diff
@@ -27,16 +27,17 @@ clean:
 # Takes approximately 7min
 setup_gke: clean
 	@cd infrastructure/terraform && \
-		terraform init && \
```

Comment:
@pankajastro on line 1, I suggest we remove `teardown_gke` for now. We'll need to wait for the job to be completed before tearing down the cluster.

Reply:
Removed